tooling/build/scripts/generate-sitemap.js

```diff
@@ -25,15 +25,15 @@ const getSchemaJson = async (filePath) => {
   try {
     const schemaContent = await fs.readFile(filePath, "utf8")
     return JSON.parse(schemaContent)
-  } catch (error) {
+  } catch {
```

Comment:
nit: instead of just silently omitting, could we `console.log` since it might be useful for debugging from logs later?